lms/djangoapps/discussion/plugins.py

```diff
@@ -34,6 +35,8 @@ class DiscussionTab(TabFragmentViewMixin, EnrolledTab):
 
     @classmethod
     def is_enabled(cls, course, user=None):
+        if not DiscussionsConfiguration.is_enabled(context_key=course.id):
```

Comment:
What will happen in case of LTI discussion provider?

Reply:
DiscussionsLTI uses seperate method to check if the tab is enabled or not.